src/app/models/buildings/building.ts

```diff
@@ -3,7 +3,7 @@ import { Game } from '../../game'
 import { MessageType, Resource } from '../types'
 
 export class Building {
-  protected level = 0
+  level = 0
```

Comment:
so as earlier, if we don't set this outside the Building, then should be protected and use getLevel